manifest/manifest.py

```diff
@@ -46,6 +46,11 @@ def itertypes(self, *types):
             for path, tests in sorted(iteritems(self._data[item_type])):
                 yield item_type, path, tests
 
+    def iterpath(self, path):
+        for type_tests in self._data.values():
+            for test in type_tests.get(path, set()):
+                yield test
```

Comment:
I know it's a simple method, but because it's an integration point, I think this would benefit from a unit test. Mind adding one?